snmp_cisco/README.md

```diff
@@ -0,0 +1,29 @@
+# Agent Check: snmp_cisco
+
+## Overview
+
+Cisco is the worldwide leader in IT, networking, and cybersecurity solutions. Install the integration to monitor all of your Cisco appliances, including routers, switches, voice gear, security appliances and more.
+
+Collect SNMP metrics from Cisco appliances, including:
+
+- Cisco Catalyst
+- Cisco Adaptive Security Appliance (ASA)
+- Cisco Meraki (note: additional events can be collected from Meraki through the [Meraki integration tile][1])
+- Cisco Nexus
+- Cisco ICM
+- Cisco ISR
+- Cisco UC Virtual Machines
```

Comment:
My vote is for no, only because we would be linking to the github repo, and on occasion that is out of sync with what's been officially shipped/released with the agent, and have seen customers complain that what they're seeing does not match what's in the repo. By not linking to the source code we give ourselves a little more room to control what customers expect